resources/function-controller/README.md

```diff
@@ -22,33 +23,28 @@ Follow the steps to install the chart:
 
 | Variable        | Description | Sample value | 
 | --------------- | ----------- | --------|
-| **FN_REGISTRY**   | The URL of the container registry Function images will be pushed to. Used for authentication.  | `https://gcr.io/` for GCR, `https://index.docker.io/v1/` for Docker Hub|
+| **FN_REGISTRY**   | The URL of the container registry Function images will be pushed to. Used for authentication.  | `https://gcr.io/` for GCR, `https://index.docker.io/v2/` for Docker Hub|
 | **FN_REPOSITORY** | The name of the container repository Function images will be pushed to. | `gcr.io/my-project` for GCR, `my-user` for Docker Hub |
 
-2. Install Knative Build:
+2. Apply crds:
+
     ```bash
-    helm install knative-build-init \
-                 --namespace="knative-build" \
-                 --name="knative-build-init" \
-                 --tls
-    
-    helm install knative-build \
-                 --namespace="knative-build" \
-                 --name="knative-build" \
-                 --tls
+    kubectl apply -f dev/crds
     ```
+
 3. Install the Function Controller charts:
+
     ```bash
     NAME=function-controller
     NAMESPACE=serverless-system
```

Comment:
ATM chart readme and project readme are inconsistent as it comes to namespace in which config maps containing Dockerfiles are created. Since those config maps must be in the same namespace in which the function may be created, it would be great to align it or at least document it to avoid confusion.